alsa-lib-gpdpocket/PKGBUILD

```diff
@@ -26,7 +26,7 @@ prepare() {
   cd "$srcdir/$pkgname-$pkgver"
   
   # replace ucm files with GPD Pocket patched files
-  cp -f ../chtrt5645.conf src/conf/ucm/chtrt5645/chtrt5645.conf
+  cp -f ../chtrt5645.conf src/conf/ucm/chtrt5645/chtrt5645.co nf
```

Comment:
?

Reply:
Oops, just saw that you already fixed this. Can't seem to retract the review though.